service_storagefacility/app/models/event/move/MoveEvents.scala

```diff
@@ -73,13 +89,17 @@ case class MoveNode(
     eventType: EventType,
     from: Option[StorageNodeDatabaseId],
     to: StorageNodeDatabaseId
-) extends MoveEvent
+) extends MoveEvent {
+
+  override val objectType: ObjectType = Node
```

Comment:
This is necessary to make the deeper levels of ugly Dao code work as expected. Not necessarily code smell since it isn't something any consumer of API's (external or internal) are exposed to.